modules/EvseManager/EvseManager.cpp

```diff
@@ -19,11 +21,16 @@ void EvseManager::ready() {
     });
 
     r_powermeter->subscribe_powermeter([this](json p) {
+        // Inform charger about current charging current. This is used for slow OC detection.
```

Comment:
Is current changging current a typo?

Reply:
No, english words just have too many meanings.